### PR TITLE
[FLINK-32731][e2e] Fix NameNode uses port 9870 in hadoop3

### PR DESCRIPTION
**Pull request by fsk119**

### CODE REVIEW

File: flink-end-to-end-tests/flink-end-to-end-tests-hive/src/test/java/org/apache/flink/tests/hive/containers/HiveContainers.java [hunk 1]
```diff
@@ -0,0 +1,248 @@
...
+
+        @Override
+        protected int getNameNodeWebEndpointPort() {
+            return NAME_NODE_WEB_PORT;
+        }
+    }
+
+    private static class Hive3Container extends HiveContainer {
+
+        private static final String METASTORE_LOG_PATH = "/tmp/root/hive.log";
```
Comment (luoyuxia) [inline flink-end-to-end-tests/flink-end-to-end-tests-hive/src/test/java/org/apache/flink/tests/hive/containers/HiveContainers.java:211]: Can these common properties be extract the parent class?

**Review by luoyuxia — COMMENTED** [review]
@fsk119 Thanks for the pr.  Only left one comment.
Btw, can you verify it works on Hive3 with our ci?
You can append a commit that change the ci profile to check it sucess again,
and then revert the commt.
You can copy my previous commit:
https://github.com/apache/flink/pull/22732/commits/98b84ec627c746df4e740c9159fbb13033f4ff2d

File: flink-end-to-end-tests/flink-end-to-end-tests-hive/src/test/java/org/apache/flink/tests/hive/containers/HiveContainers.java (hunk 1 above)
Comment (fsk119) [reply flink-end-to-end-tests/flink-end-to-end-tests-hive/src/test/java/org/apache/flink/tests/hive/containers/HiveContainers.java:211]: no. Hive3 places logs in the "/tmp/root/hive.log", but hive2 puts logs in the "/tmp/hive/hive.log".

**Review by luoyuxia — APPROVED** [review]
@fsk119 Thanks for update. LGTM

**Pull request closed without merging**